schema/filter_artworks.js

```diff
@@ -145,6 +145,10 @@ function filterArtworks(primaryKey) {
         delete gravityOptions.medium;
       }
 
+      if (accessToken) {
+        return gravity.with(accessToken)('filter/artworks', gravityOptions);
+      }
+
       return gravity('filter/artworks', gravityOptions);
```

Comment:
So [the original question](https://github.com/artsy/metaphysics/pull/442#discussion_r83823511) was about removing this duplication. i.e. replacing lines 148..152 with just:

```js
return gravity.with(accessToken)('filter/artworks', gravityOptions);
```